tests/test_unclassified_job.py

```diff
@@ -12,18 +12,18 @@ class TestUnclassifiedJobs:
     @pytest.mark.nondestructive
     def test_unclassified_failure(self, base_url, selenium):
         # Open resultset page and search for next unclassified failure
-        resultset_page = ResultsetPage(base_url, selenium).open()
-        assert resultset_page.unclassified_failure_count > 0
+        treeherder_page = TreeherderPage(base_url, selenium).open()
```

Comment:
I think you can get away with just calling this variable `page`. I've taken to doing this when there's only one page object in use.